tests/server.spec.js

```diff
@@ -619,7 +614,7 @@ async function expectListResponseContains(path, object) {
 
 function expectStatus(response, expectedCode, type = "application/json") {
   if (response.status === 400 && expectedCode != 400) {
-    expect(response.body).toHaveProperty("errors", "");
+    expect(response.body).not.toHaveProperty("errors", "");
```

Comment:
Why are we checking that we don't have the `errors` field in the response here?

Reply:
Oops that's a leftover from some debugging session, nice catch!